src/components/Trip/TripDialogMobileContent.jsx

```diff
@@ -21,21 +18,24 @@ const makeMapStyles = ({ toolbarHeight }) => ({
   }
 })
 
-const TripLayout = () => {
-  const { trip } = useTrip()
+const TripDialogMobileContent = ({ titleRef }) => {
+  const [toolbarHeight, setToolbarHeight] = useState(0)
 
-  const toolbarHeight = document.getElementById('coz-bar').offsetHeight
   const toolbarProps = useMemo(() => ({ height: toolbarHeight }), [
     toolbarHeight
   ])
-  const title = useMemo(() => getEndPlaceDisplayName(trip), [trip])
-  const styles = useMemo(() => makeMapStyles({ toolbarHeight }), [
-    toolbarHeight
-  ])
+
+  const styles = useMemo(
+    () => makeMapStyles({ toolbarHeight: toolbarHeight }),
+    [toolbarHeight]
+  )
+
+  useEffect(() => {
+    setToolbarHeight(titleRef?.current?.offsetHeight)
```

Comment:
Afin de s'assurer que `toolbarHeight` soit toujours de type `number`
```suggestion
useEffect(() => {
  if (titleRef?.current) {
    setToolbarHeight(titleRef.current.offsetHeight)
  }
}, [titleRef])
```